.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,79 @@
+name: Auto-Merge Dependabot PRs (minor/patch only)
+
+on:
+    pull_request:
+        types:
+            - labeled # Triggered when a label is added to a pull request
+
+jobs:
+    auto-merge:
+        if: github.event.pull_request.user.login == 'dependabot[bot]' # Ensures this only runs for Dependabot PRs
+        runs-on: ubuntu-latest
+        env:
+            JOB_ABORTED: 'false' # env var used to indicate whether job should be aborted
+        steps:
+            - name: Check if the PR has the 'dependencies' label
+              uses: actions/github-script@v6
+              id: check-label
+              with:
+                  script: |
+                      const { labels } = context.payload.pull_request;
+
+                      // When dependabot creates PRs, it adds 'dependencies' as a label to the PRs.
+                      // Here we check to see if 'dependencies' label is present
+                      const hasDependenciesLabel = labels.some(label => label.name === 'dependencies');
+
+                      if (!hasDependenciesLabel) {
+                        console.log("The 'dependencies' label is missing. Auto-merge aborted.");
+                        core.exportVariable('JOB_ABORTED', 'true');
```

Comment:
Is `core` available by default in our workflows?

Reply:
`actions/github-script@v6` brings in core by default